assets/fonts/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Wrong license: https://github.com/googlefonts/noto-fonts/blob/master/LICENSE

The Apache License is outdated since 30.09.2015.

Reply:
I replaced it with the correct license.